hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -656,7 +659,7 @@ private FlinkOptions() {
   public static final ConfigOption<String> CLUSTERING_PLAN_PARTITION_FILTER_MODE_NAME = ConfigOptions
       .key("clustering.plan.partition.filter.mode")
```

Comment:
Can we also 
1. rename the param `payload.ordering.field` to `precombine.field`;
2. flip `write.ignore.failed` default to be false.